gqlex.g4

```diff
@@ -40,19 +76,19 @@ pathExpression
     ;
 
 rootPathExpression
-    : (ALL | NODE) (pathElement (NODE pathElement)*)+
+    : (DOUBLE_SLASH | SLASH) (pathElement (SLASH pathElement)*)?
     ;
 
 rangePathExpression
-    : RANGE_OPEN (INT? COLON INT?) RANGE_CLOSE rootPathExpression
+    : RANGE_OPEN (rangeStartIntValue? COLON rangeEndIntValue?) RANGE_CLOSE rootPathExpression
     ;
 
 elementSelectionExpression
-    : lValue EQUALS rValue
+    : nameValue EQUALS nameValue
     ;
 
 pathElement
-    : lValue (LBRACK elementSelectionExpression RBRACK)? (LPAREN conditionExpression RPAREN)?
+    : nameValue (LBRACK elementSelectionExpression RBRACK)? (LPAREN conditionExpression RPAREN)?
```

Comment:
I suggest to rename conditionOperator to comparisonOperator.

Reply:
Done